### PR TITLE
OKTA-90565

### PR DESCRIPTION
**Pull request by larsjohansen-okta**
## Description:
- Added suppport for token_type_hint for token introspection and revocation
- https://github.com/okta/okta-core/pull/21393

## Type of Pull Request:

- [x] Content (Documentation updates or typo-fixes)
- [ ] Blog Post
- [ ] Functional (CSS changes, JS updates, or layout modifications)

### Resolves:
* [OKTA-90565](https://oktainc.atlassian.net/browse/OKTA-90565)

## How Has This Been Tested?
- [ ] I have built this locally and verified that it does not break existing functionality.
- [ ] For functional changes, I have tested against Chrome, Firefox, and Safari, and mobile.
- [ ] For functional changes, I have added tests.

## Primary Reviewers:
- @mystiberry-okta @petermiller-okta

### CODE REVIEW

File: _source/_docs/api/resources/oidc.md [hunk 1]
```diff
@@ -498,7 +498,7 @@ The following parameters can be posted as a part of the URL-encoded form values
...
 | token                 | An Access Token or Refresh Token.                                                                                                                                                                                                                                         | String |
-| token_type_hint       | A hint of the type of `token`.                                                                                                                                                                                                                                            | String |
+| token_type_hint       | A hint of the type of `token`. Valid values are `access_token` and `refresh_token`.                                                                                                                                                                                                                                         | String |
```
Comment (cwu-okta) [inline _source/_docs/api/resources/oidc.md]: Isnt' this now an enum type?

File: _source/_docs/api/resources/oidc.md [hunk 2]
```diff
@@ -359,7 +359,7 @@ The following parameters can be posted as a part of the URL-encoded form values
...
 | token                 | An Access Token, ID Token, or Refresh Token.                                                                                                                                                                                                                | String |
-| token_type_hint       | A hint of the type of *token*.                                                                                                                                                                                                                              | String |
+| token_type_hint       | A hint of the type of `token`. Valid values are `access_token`, `id_token` and `refresh_token`.                                                                                                                                                                                                                            | String |
```
Comment (cwu-okta) [inline _source/_docs/api/resources/oidc.md]: Isn't this now an enum type?

Comment (larsjohansen-okta) [inline _source/_docs/api/resources/oidc.md]: Good catch!

File: _source/_docs/api/resources/oidc.md (hunk 1 above)
Comment (larsjohansen-okta) [inline _source/_docs/api/resources/oidc.md]: Good catch!

**Pull request merged**